onnxscript/function_libs/torch_lib/registration.py

```diff
@@ -74,7 +74,8 @@ def torch_op(
     """Register a torch op.
 
     Args:
-        name: ATen name of the function. E.g. "aten::add".
+        name: Qualified ATen name of the function. E.g. "aten::relu", "aten::add.Tensor".
```

Comment:
`name Union[str, Tuple[str, ...]]` to annotate this function?

Reply:
Thanks, just added